src/includes/class-health-check-debug-data.php

```diff
@@ -392,6 +397,11 @@ static function debug_data( $locale = null ) {
 			'label' => __( 'PHP SAPI', 'health-check' ),
 			'value' => ( ! function_exists( 'php_sapi_name' ) ? __( 'Unable to determine PHP SAPI', 'health-check' ) : php_sapi_name() ),
 		);
+		$info['wp-server']['fields'][] = array(
+			'label'   => __( 'PHP User', 'health-check' ),
+			'value'   => get_current_user() . ' (' . getmyuid() . ')',
```

Comment:
I had a thought here, it's not unlikely that these kind of functions might be disabled on a host level in various scenarios, we should add a `function_exists` wrapper to this check to ensure we don't call a fatal error in such cases.

Reply:
I wrapped it in a test for the two functions involved.